index.html

```diff
@@ -6,15 +6,173 @@
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Libre+Baskerville:ital@0;1&display=swap" rel="stylesheet">
     <!-- Add a link to your custom CSS here -->
 
     <title>Responsive grid project</title>
 </head>
 
 <body>
+    <!-- Header -->
+    <header class="container">
+        <div class="row">
+            <nav class="navbar navbar-expand-sm bg-white navbar-light fixed-top">
+               
+                <div class="col-12 col-sm-8 order-sm-2 justify-content-around d-flex flex-sm-column align-items-sm-end"> 
+                    <a class="col-8 col-sm-5 col-md-4 col-lg-2 appointment-button bg-lightmint text-decoration-none text-center" href="#">Book an Appointment</a>
 
-    <!-- Add your markup here -->
+                    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#collapsibleNavbar">
+                         <span class="navbar-toggler-icon"></span>
+                    </button>
+ 
+                    <div class="collapse navbar-collapse justify-content-end fw-bolder" id="collapsibleNavbar">
+                        <ul class="navbar-nav">
+                            <li class="nav-item">
+                                <a class="nav-link" href="#home">Home</a>
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link" href="#">About</a>
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link" href="./services.html">Services</a>
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link" href="#">Blog</a>
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link" href="./contact.html">Contact</a>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="col-12 col-sm-4 order-sm-1 logo mt-2"> 
+                    <a class="navbar-brand title fst-italic fs-6 py-3" href="#">Maria Hope, Ph.D. -Psychologist</a>
+                </div>
 
+            </nav>
+        </div>
+    </header>
+    
+    <!-- Main -->
+    <main id="home">
+        <!-- Hero Section -->
+        <section class="container-fluid my-5">
+            <div class="row my-5 text-center hero">
+                <div class="col-12 col-sm-6 order-sm-2 p-0 ">
+                    <img class="img-fluid w-100 h-100" src="./img/first image.webp" alt="Hero Section Image">
+                </div>
+                <div class="col-12 col-sm-6 order-sm-1 bg-lightmint d-flex flex-column justify-content-center ">
+                    <h1 class="fw-lighter lh-base my-3">
+                        Online Therapy for Emotional Health
+                    </h1>
```

Comment:
Don't rely too much on Bootstrap for aesthetics, fonts, sizing, spacing etc. Focus on using Bootstrap for its grid system only (containers, rows and columns) and write your own CSS for everything else. 

Here for example, you could set the font-size to be bigger on mobile by default (34px in the design), and then grow it to 49px on desktop using a media query in your CSS.